main.lua

```diff
@@ -51,13 +51,24 @@ function thm:getIndex(t, v)
 end
 
 function thm:blacklistPlayer(player)
-	local index = thm:getIndex(DCTHM.blacklist, player)
-	if (index > 0) then
-		table.remove(DCTHM.blacklist, index)
-		DEFAULT_CHAT_FRAME:AddMessage("|cff00D1FFTHM:|r " .. player .. " no longer blacklisted")
+	
+	-- If no name entered for blacklist command, print current blacklist
+	if player == nil then
+		DEFAULT_CHAT_FRAME:AddMessage("|cff00D1FFTHM:|r ~ Current Blacklist ~")
+		for index, blplayer in ipairs(blacklist) do
+			DEFAULT_CHAT_FRAME:AddMessage("|cff00D1FFTHM:|r " .. blplayer)
+		end
+		

```

Comment:
1) Should this print out  really be on multiple lines?
2) Should this be sorted to easier spot players?
3) The whole for loop can be replaced with `table.concat`:

Reply:
Oh my goodness, I also have never used github so I didnt even realise there was comments here!
Hmmmm good questions.
1&2:
If it's printed on multiple lines it makes it fairly easy to scan and therefore doesn't really need to be sorted.
If it's all on one line then having it sorted (probably alphabetically?) would help you look for a particular name.  This might be the better option.
3:
That is a much more elegant way of doing it.